src/libraries/System.Drawing.Common/tests/GdiplusTests.cs

```diff
@@ -16,5 +18,28 @@ public void IsAtLeastLibgdiplus6()
         {
             Assert.True(Helpers.GetIsWindowsOrAtLeastLibgdiplus6());
         }
+
+        [Fact]
+        public void GdiplusDefaultApiVersion()
```

Comment:
It would better to add a regression test for the bug that this is fixing.

Testing trivial internal implementation details via private reflection is not very useful.

Reply:
Ok, sounds good. I'll add that test.

Reply:
Updated, thanks for the feedback.